csp-mixin/panels.libsonnet

```diff
@@ -842,5 +842,312 @@ local commonlib = import 'common-lib/common/main.libsonnet';
           },
         },
       ]),
+
+    // Azure Virtual Machines
+
+    avm_instance_count:
+      this.signals.virtualMachinesOverview.instanceCount.asStat()
+      + commonlib.panels.generic.stat.base.stylize(),
+
+    avm_availability:
+      this.signals.virtualMachinesOverview.vmAvailability.common
+      + commonlib.panels.generic.stat.base.new(
+        'VM Availability',
+        [
+          this.signals.virtualMachinesOverview.vmAvailability.asTarget()
+          + g.query.prometheus.withFormat('time_series')
+          + g.query.prometheus.withInstant(true)
+          + g.query.prometheus.withRange(false),
+        ],
+        'Measure of Availability of Virtual machines'
+      )
+      + g.panel.stat.options.withColorMode('background')
+      + g.panel.stat.standardOptions.withUnit('short')
+      + g.panel.stat.standardOptions.withOverrides([
+        {
+          matcher: {
+            id: 'byType',
+            options: 'number',
+          },
+          properties: [
+            {
+              id: 'mappings',
+              value: [
+                {
+                  options: {
+                    '0': {
+                      color: 'red',
+                      index: 1,
+                      text: 'Not Available',
+                    },
+                    '1': {
+                      color: 'green',
+                      index: 0,
+                      text: 'Available',
+                    },
+                  },
+                  type: 'value',
+                },
+              ],
+            },
```

Comment:
nit:you can define valueMapping in the signal itself. Check signals lib Readme.

Reply:
I tried to config **valueMappings** on the signal but this is what I see:

<img width="658" alt="Screenshot 2024-09-20 at 14 44 33" src="https://github.com/user-attachments/assets/f863b3ec-2a5e-49f7-ad75-6cb623dd94ec">

This is the expected behaviour:
<img width="653" alt="Screenshot 2024-09-20 at 14 45 48" src="https://github.com/user-attachments/assets/458ac0b0-afda-443a-9383-9265b48206ef">

No idea what is doing behind.

Reply:
got it. You can always check json of the generated panel to see if everything is rendered as expected.